src/test/java/net/openhft/chronicle/wire/AbstractUntypedFieldTest.java

```diff
@@ -49,6 +51,20 @@ void untypedFieldsShouldBeNull(Function<Bytes<byte[]>, Wire> wireConstruction) {
         assertNull(holder.a);
     }
 
+    @ParameterizedTest
+    @MethodSource("provideWire")
+    void missingAliasExceptionsShouldNotBeSwallowed(Function<Bytes<byte[]>, Wire> wireConstruction) {
+        final Bytes<byte[]> bytes = Bytes.from("!net.openhft.chronicle.wire.AbstractUntypedFieldShouldBeNull$Holder {\n" +
+                "  a: !MissingAlias {\n" +
+                "  }\n" +
+                "}");
+        final Wire textWire = wireConstruction.apply(bytes);
+
+        assertThrows(MissingClassOrClassAliasException.class, () -> {
+            textWire.getValueIn().object(Holder.class);
```

Comment:
so current behaviour is for the exception to be silently swallowed?

Reply:
Yeah but only due to that catch block that I put on [the issue](https://github.com/OpenHFT/Chronicle-Wire/issues/424). I don't think it's intentional.